v22.1/cockroach-cert.md

```diff
@@ -3,121 +3,44 @@ title: cockroach cert
 summary: A secure CockroachDB cluster uses TLS for encrypted inter-node and client-node communication.
 toc: true
 key: create-security-certificates.html
+filter_category: security_cert
+filter_html: Use cockroach cert
+filter_sort: 1
 docs_area: reference.cli
 ---
 
-The CockroachDB CLI's `cockroach cert` command allows you to generate [private key/public certificate pairs for TLS authentication and encryption in communication between CockroachDB nodes, and from SQL clients to the cluster.
+To secure your CockroachDB cluster's inter-node and client-node communication, you need to provide a Certificate Authority (CA) certificate that has been used to sign keys and certificates (SSLs) for:
 
-To learn more, explore:
+- Nodes
+- Clients
+- DB Console (optional)
 
-- [Transport Layer Security (TLS) and Public Key Infrastructure (PKI)](security-reference/transport-layer-security.html)
-- [Using the CockroachDB CLI to provision a development cluster](manage-certs-cli.html).
-- [Managing CockroachDB security certificates with HashiCorp Vault](manage-certs-vault.html).
+To create these certificates and keys, use the `cockroach cert` [commands](cockroach-commands.html) with the appropriate subcommands and flags, use [`openssl` commands](https://wiki.openssl.org/index.php/), or use a [custom CA](create-security-certificates-custom-ca.html) (for example, a public CA or your organizational CA).
```

Comment:
```suggestion
To create these certificates and keys, choose **one** of the following options:

- Use the `cockroach cert` [commands](cockroach-commands.html) with the appropriate subcommands and flags.
- Use [`openssl` commands](https://wiki.openssl.org/index.php/).
- Use a [custom CA](create-security-certificates-custom-ca.html) (for example, a public CA or your organizational CA).
```

This is a really complex sentence that feels like a run-on even though it isn't. Turning it into bullets might make it more readable.